net-im/discord-bin/discord-bin-0.0.20-r2.ebuild

```diff
@@ -21,41 +21,47 @@ SRC_URI="https://dl.discordapp.net/apps/linux/${MY_PV}/${MY_PN}-${MY_PV}.tar.gz"
 LICENSE="all-rights-reserved"
 SLOT="0"
 KEYWORDS="amd64"
+RESTRICT="bindist mirror strip test"
+IUSE="+seccomp swiftshader system-ffmpeg"
 
-# libXScrnSaver is used through dlopen (bug #825370)
```

Comment:
Shouldn't this be an `optfeature` then?

Reply:
I'm not sure. In the Teams ebuild, it's just listed, so I went with that.